docs/modules/ROOT/pages/utilities.adoc

```diff
@@ -0,0 +1,43 @@
+= Utilities
+
+The OpenZeppelin Stylus Contracts provide a ton of useful utilities that you can use in your project.
+For a complete list, check out the https://docs.rs/openzeppelin_stylus/utils/index.html[API Reference].
+Here are some of the more popular ones.
+
+[[introspection]]
+== Introspection
+
+As well as in Solidity, at Stylus, it's frequently helpful to know whether a contract supports an interface you'd like to use.
+https://eips.ethereum.org/EIPS/eip-165[`ERC-165`] is a standard that helps do runtime interface detection.
+Contracts provide helpers both for implementing ERC-165 in your contracts and querying other contracts:
```

Comment:
Slight rephrase, and I don't think we are providing support for checking interfaces right?
```suggestion
Contracts for Stylus provides helpers for implementing ERC-165 in your contracts:
```

Reply:
Yeah. Just use `Contracts for Stylus provide` without "s". Since it has already plural form

Reply:
This is a nit. But Actually it is a singular entity even if it ends with "s"

> Grammatical correctness: "OpenZeppelin Contracts for Stylus" is treated as a singular entity, representing a single library or project. Therefore, the singular verb "exists" is the correct form to use

Reply:
So be it. But I think there're many artifacts of "mine approach" around the codebase :)